Extension/src/Debugger/configurationProvider.ts

```diff
@@ -662,3 +670,112 @@ export class ConfigurationSnippetProvider implements vscode.CompletionItemProvid
         return Promise.resolve(new vscode.CompletionList(items, true));
     }
 }
+
+export function buildAndDebugActiveFileStr(): string {
+    return `${localize("build.and.debug.active.file", 'Build and debug active file')}`;
+}
+
+export async function buildAndDebugShortCut(textEditor: vscode.TextEditor, cppVsDbgProvider: CppVsDbgConfigurationProvider | null, cppDbgProvider: CppDbgConfigurationProvider, debugModeOn: boolean = true): Promise<void> {
```

Comment:
Function names should be a verb or verb phrase.  Can you remove "ShortCut" from the name?